sdk/cosmosdb/cosmos/src/queryExecutionContext/defaultQueryExecutionContext.ts

```diff
@@ -216,6 +232,17 @@ export class DefaultQueryExecutionContext implements ExecutionContext {
           responseHeaders[Constants.HttpHeaders.QueryMetrics]["0"] = queryMetrics;
         }
 
+        if (operationOptions && operationOptions.ruCapPerOperation && ruConsumedManager) {
+          ruConsumedManager.addToRUConsumed(getRequestChargeIfAny(responseHeaders));
+          if (ruConsumedManager.getRUConsumed() > operationOptions.ruCapPerOperation) {
```

Comment:
Should this be `>=`?

Reply:
No, we decided go with strictly greater value.